home/ivp-planning/machines/scale-objects.md

```diff
@@ -3,11 +3,11 @@
 The scaling objects offers a way to alter the size of every object. This is useful to make adjustments of objects that are not [customizable](customizable-machines.md).
 
 {% hint style="warning" %}
-Use the scaling function with care to avoid deformations or glitches. If it is possible to use a [customization ](customizable-machines.md)option of the object, always use the option.
+Use the scaling function with care to avoid deformations or glitches. If it is possible to use a [customization](customizable-machines.md) option of the object, always use the option.
 {% endhint %}
 
 {% hint style="info" %}
-Currently it is only possible to scale objects along the horizontal axis. A scaling along all axis is currently in development.
+Use the scaling function with care to avoid deformations or glitches. If it is possible to use a [customization](customizable-machines.md) option of the object, always use the option.
```

Comment:
This section is now duplicated and the original info text here is missing. Bring back the original text, but adjust the wording:

* "horizontal axis" should be Z-Axis
* We do support scaling on the Z-Axis for some objects, just not for most
* Remove the part about it currently being in development